examples/gnrc_border_router/Kconfig

```
@@ -1,10 +1,10 @@
-if MODULE_ETHOS
+if USEMODULE_ETHOS
 config GNRC_DHCPV6_CLIENT_6LBR_STATIC_ROUTE
     default y
-    depends on MODULE_GNRC_DHCPV6_CLIENT_6LBR && KCONFIG_MODULE_GNRC_DHCPV6
+    depends on USEMODULE_GNRC_DHCPV6_CLIENT_6LBR && KCONFIG_USEMODULE_GNRC_DHCPV6
```

Should it rather be:
```suggestion
    depends on USEMODULE_GNRC_DHCPV6_CLIENT_6LBR && USEMODULE_GNRC_DHCPV6
```
(which means there's an issue in master).

---

No, `KCONFIG_USEMODULE_GNRC_DHCPV6` is a dependency so the value is only applied when the configuration via Kconfig for that module is enabled.